bots/github-trigger

```diff
@@ -5,24 +5,22 @@ import sys
 
 sys.dont_write_bytecode = True
 
-from common import testinfra
```

Comment:
This commit is mostly moving around existing code (with some obvious adjustments for moved paths/modules), so I didn't review the actual code in great detail.